tests/test_enterprise/api/test_views.py

```diff
@@ -7397,9 +7398,132 @@ def test_list_learner_statuses(self):
         )
         response = self.client.get(url)
         assert response.json().get('count') == 2
-        statuses = [result.get('member_status') for result in response.json().get('results')]
+        statuses = [result.get('status') for result in response.json().get('results')]
         assert statuses.sort() == ['accepted', 'pending'].sort()
 
+    def test_list_learners_bad_sort_by(self):
+        """
+        Test that the list learners endpoint properly validates sort by query params
+        """
+        url = settings.TEST_SERVER + reverse(
+            'enterprise-group-learners',
+            kwargs={'group_uuid': self.group_1.uuid},
+        ) + "?sort_by=ayylmao"
+
+        response = self.client.get(url)
+        assert response.status_code == 400
+        assert response.data.get('sort_by')
+
+        long_query_string = "foo".join('bar' for _ in range(320))
+        url = settings.TEST_SERVER + reverse(
+            'enterprise-group-learners',
+            kwargs={'group_uuid': self.group_1.uuid},
+        ) + f"?user_query={long_query_string}"
+        response = self.client.get(url)
+        assert response.status_code == 400
+        assert response.data.get('user_query')
+
+    def test_list_learners_filtered(self):
```

Comment:
It'd be good to have an additional test in here to protect against any future changes to de-parameterize the raw sql.  Like pass in some Little Bobby Tables type of user query.
https://xkcd.com/327/

Reply:
done!